AutomatedReview/PeriodicPipelineGuide.md

```diff
@@ -0,0 +1,51 @@
+# Periodic pipeline Guide
+
+Periodic pipeline is used to cover the additional platforms and build configurations that are not built in Automated Review pipeline. Though periodic builds are not as critical as AR builds, they provide useful information to ensure the quality of O3DE.
+
+## Periodic Incremental Daily pipeline
+
+Periodic incremental daily pipeline runs every day at 10pm PST, it runs incremental builds and reuses workspace from previous build.
+
+Pipeline URL: https://jenkins.build.o3de.org/job/O3DE_periodic-incremental-daily/
+
+### Build Platforms and Configuration:
```

Comment:
Should we put any details about how this is configured in `build_config.json`? I would think someone looking to add more jobs to this may need a reminder.

Reply:
Yeah, there is another guide for modifying Jenkins pipeline https://github.com/o3de/sig-build/blob/main/AutomatedReview/JenkinsPipelineGuide.md, I will include this link here.